oc_ldap_user_locker/locker.py

```diff
@@ -142,6 +140,48 @@ def _compare_attribute(self, values, match_conf):
         logging.debug("Finall check: returning '%s'" % str(_result))
         return _result
 
+    def _compare_attribute(self, attrib, user_rec, match_conf):
+        """
+        Get values of the attribute and call the method to compare them
+        :param attrib: attribute to compare
+        :param OcLdapRecord user_rec: LDAP record for user account
+        :param dict match_conf: configuration dictionary
+        """
+
+        if not match_conf:
+            # sure it is a bug
+            raise ValueError("No match configuration given")
+
+        if "." not in attrib:
+            _values = user_rec.get_attribute(attrib)
+            return self._compare_attribute_values(_values, match_conf)
+        else:
```

Comment:
We do not need `else` here since we are doing `return` in `if`.
Thus we may increase code readability by doing all the things below as usual without extra indent.